SEUNAGBEYE/Math-Skill-Game
# #160831444 Add the Answer component

### What does this PR do?
Creates the Answer component

### Description of Task to be completed?
-add the answer component
-add props type validation

## How should this be manually tested?
- Fetch this branch with `git fetch origin ft-add-answer-component-160831444:ft-add-answer-component-160831444`
- Checkout to `ft-add-answer-component-160831444`
- Run `yarn start`
- The app should be started, then visit the webpage and you should see the `Answers Icon`

### Any background context you want to provide?
NA


### What are the relevant pivotal tracker stories?
[#160831444](https://www.pivotaltracker.com/n/projects/2200608/stories/160831444)

### Screenshots (if appropriate)
NA
